submissions/oluwaboriShobayo/databases/init.sql

```diff
@@ -0,0 +1,151 @@
+CREATE DATABASE StylebyBori;
+SHOW DATABASES;
+USE StylebyBori;
+
+-- This is the products table that references the admin and categories table
+CREATE TABLE PRODUCTS (
+		id INT NOT NULL AUTO_INCREMENT,
+		name VARCHAR(100) NOT NULL,
+		description VARCHAR(255) NOT NULL,
+		image BLOB,
+		unit_price DECIMAL (10,2),
+		stock_level VARCHAR(50),
+        created_by INT NOT NULL,
+        categories_id INT NOT NULL,
```

Comment:
When I run your query, I got an error, It was trying to look for categories table, meanwhile, categories table haven't been created at this point. Kindly rearrange your query orderly as it comes